### PR TITLE
Candidates bios

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
CC: @aloisreitbauer, @ccaraman, @skateordie, @freeformz, @irabinovitch, @kbychu, @lizthegrey, @lmineiro, @markcartertm, @mtwo, @StevenLeRoux, @tedsuo

### CODE REVIEW

File: community-members.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ Repo: [open-telemetry/opentelemetry-js](https://github.com/open-telemetry/opente
 Approvers:
 
 - [Yuri Shkuro](https://github.com/yurishkuro), Uber
+- [Isobel Redelmeier](https://github.com/iredelmeier), LightStep
```
Comment (bhs) [inline community-members.md]: (please remove these other files from the PR)

Comment (SergeyKanzhelev) [reply community-members.md]: merge incident. Fixed

**Comment by lizthegrey** [comment]
Please use my bio from lizthegrey.com; I'll have a separate candidate statement later :)

**Comment by SergeyKanzhelev** [comment]
@lizthegrey so you want the bio to be replaced with this:

Liz is a developer advocate, labor and ethics organizer, and Site Reliability
Engineer (SRE) with 15+ years of experience. She is an advocate at
[Honeycomb.io](https://www.honeycomb.io/) for the SRE and Observability
communities, and previously was an SRE working on products ranging from the
Google Cloud Load Balancer to Google Flights.

She lives in Brooklyn with [her wife Elly](https://elly.town/), metamours, and
[a Samoyed/Golden Retriever mix](http://elly.dog/), and in San Francisco and
Seattle with her other partners. She plays classical piano, leads an [EVE Online
alliance](https://www.of-sound-mind.com/), and advocates for transgender rights
as a board member of the [National Center for Transgender
Equality](https://transequality.org/).

**Comment by lizthegrey** [comment]
> @lizthegrey so you want the bio to be replaced with this:
> 
> Liz is a developer advocate, labor and ethics organizer, and Site Reliability
> Engineer (SRE) with 15+ years of experience. She is an advocate at
> [Honeycomb.io](https://www.honeycomb.io/) for the SRE and Observability
> communities, and previously was an SRE working on products ranging from the
> Google Cloud Load Balancer to Google Flights.
> 
> She lives in Brooklyn with [her wife Elly](https://elly.town/), metamours, and
> [a Samoyed/Golden Retriever mix](http://elly.dog/), and in San Francisco and
> Seattle with her other partners. She plays classical piano, leads an [EVE Online alliance](https://www.of-sound-mind.com/), and advocates for transgender rights
> as a board member of the [National Center for Transgender Equality](https://transequality.org/).

First para is fine by itself but yes. Thamks

**Comment by SergeyKanzhelev** [comment]
Done: https://github.com/open-telemetry/community/pull/225/commits/95f3fe97a919d23fce4e6e5e528550f8cbab77c0

**Pull request merged**